pkg/storage/bloom/v1/test_util.go

```diff
@@ -74,8 +75,16 @@ func MkBasicSeriesWithBlooms(nSeries, keysPerSeries int, fromFp, throughFp model
 			it := tokenizer.Tokens(fmt.Sprintf("series %d", i*keysPerSeries+j))
 			for it.Next() {
 				key := it.At()
+				// series-level key
 				bloom.Add(key)
 				keys = append(keys, key)
+
+				// chunk-level key
+				for _, chk := range series.Chunks {
+					tokenBuf, prefixLen := prefixedToken(nGramLen, chk, nil)
+					tokenBuf = append(tokenBuf[:prefixLen], key...)
+					bloom.Add(tokenBuf)
```

Comment:
This is needed to make this test pass:
https://github.com/grafana/loki/blob/ecab33a93250b0ffed3b8684b72eeb9334982b7d/pkg/bloomgateway/bloomgateway_test.go#L407